documentation-site/examples/popover/dismiss.js

```diff
@@ -7,18 +7,17 @@ import {Paragraph1} from 'baseui/typography';
 
 export default () => {
   const [useCss, theme] = useStyletron();
+  const contentCx = useCss({
```

Comment:
not related to this PR but this example was broken bc of the placement of this hook